templates/release/de/meter/cg-em24_e1_0.yaml

```diff
@@ -1,6 +1,10 @@
 product:
+  # EM24 mit Ethernet anschluss
   brand: Carlo Gavazzi
   description: EM24_E1
+description: |
+ EM24 mit Ethernet anschluss. Benutze die EM24 für die EM24 mit RS-485 anschluss, denn die Definition is nicht compatibel.
```

Comment:
EM24_E1 mit Ethernet-Anschluss. Benutze die EM24 für die EM24 mit RS-485-Anschluss, denn die Definition is nicht kompatibel.